providers/netty4/src/main/java/org/asynchttpclient/netty/handler/Processor.java

```diff
@@ -71,11 +75,41 @@ public void channelRead(final ChannelHandlerContext ctx, Object msg) throws Exce
                 ac.call();
                 Channels.setDiscard(channel);
             }
+            ReferenceCountUtil.release(msg);
+
 
         } else if (attribute instanceof NettyResponseFuture) {
             NettyResponseFuture<?> future = (NettyResponseFuture<?>) attribute;
             protocol.handle(channel, future, msg);
 
+        } else if (attribute instanceof StreamedResponsePublisher) {
```

Comment:
Is there a way to emulate a connection crash? As said, I've tried to write a test for this, but I can't make the connection crash (so that I can confirm the retry logic is run). My attempt was to call `channel.close().await()`, but that blocks indefinitely.